### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by pprathameshmore**

### CODE REVIEW

**Comment by pprathameshmore** [comment]
@poush Done! Now you can merge it!

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# monsterInMe
\ No newline at end of file
+# monsterInMe
+<h1> Monster In Me </h1>
```
Comment (poush) [inline README.md]: You added one more heading.

Comment (pprathameshmore) [reply README.md]: Should I remove this? @poush

Comment (poush) [reply README.md]: Yes please.

**Comment by pprathameshmore** [comment]
Done!

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# monsterInMe
\ No newline at end of file
+# monsterInMe
+#DEMO
```
Comment (poush) [inline README.md:2]: Add a line break after this.

**Comment by pprathameshmore** [comment]
Added line break @poush

File: README.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 # monsterInMe
-#DEMO
+#DEMO </br>
```
Comment (poush) [inline README.md]: Please don't add html one, instead follow this https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lines

File: README.md [hunk 4]
```diff
@@ -1 +1,3 @@
-# monsterInMe
\ No newline at end of file
+# monsterInMe
+#DEMO </br>
+<img src="img/gif/demo.gif" alt="Demo GIF" border="0">
```
Comment (poush) [inline README.md]: Don't add html tags here. Please use markdown.

File: README.md (hunk 3 above)
Comment (pprathameshmore) [reply README.md]: Oh! Thank you! @poush

File: README.md (hunk 4 above)
Comment (pprathameshmore) [reply README.md]: Done! Removed HTML tag and added by simply pressing enter button!

Comment (poush) [reply README.md]: I meant this image tag. There is markdown syntax to add an image.

**Comment by pprathameshmore** [comment]
Ok! I will work on this issue!

**Comment by pprathameshmore** [comment]
@poush I made changes!

**Comment by poush** [comment]
Thanks!

**Pull request merged**